.github/workflows/test.yml

```diff
@@ -148,53 +132,22 @@ jobs:
               shell: bash -l {0}
 
             - name: Start dfx
-              if: ${{ !env.IS_RELEASE_CANDIDATE }}
+              if: ${{ env.IS_FEATURE_BRANCH_PR == 'true' }}
               working-directory: ${{ matrix.tests.path }}
               run: dfx start --clean --background --host 127.0.0.1:8000 --artificial-delay 0
 
-            - name: Start dfx with artifical delay
-              if: ${{ env.IS_RELEASE_CANDIDATE }}
+            - name: Start dfx with artifcial delay
```

Comment:
artificial is spelled incorrectly, and isn't the name here wrong? Shouldn't it be swapped?